src/main/java/com/gooddata/warehouse/WarehouseService.java

```diff
@@ -262,6 +263,7 @@ private Warehouse setWarehouseConnection(Warehouse warehouse) {
         notNull(warehouse, "warehouse");
         warehouse.setWarehouseHost(warehouseHost);
         warehouse.setWarehousePort(warehousePort);
+        warehouse.setConnectionUrl(getConnectionUrlForWarehouse(warehouse));
```

Comment:
doesn't make much sense for me, whole method setWarehouseConnection() should be removed now